lms/templates/courseware/courses.html

```diff
@@ -12,6 +12,13 @@
       <%static:include path="discovery/${template_name}.underscore" />
   </script>
   % endfor
+  <script type="text/javascript">;(function (define) {{
+      define('course_discovery_meanings', function() {{
+          'use strict';
+          return ${course_discovery_meanings};
```

Comment:
This is an example where it is confusing that this was pre-escaped. This code looks very dangerous as it stands, because it isn't being escaped in any way.

See also the recent RCA about the risk of XSS attacks using the approach I suggested:

https://openedx.atlassian.net/wiki/pages/viewpage.action?spaceKey=TNL&title=RCA%3A+2015-05-26+-+Student+Profile+XSS+vulnerability

I don't know if this will be an issue for you, but you might want to consider using JSON escaping as suggested in the RCA: ``simplejson.JSONEncoderForHTML`` (http://stackoverflow.com/questions/7784303/django-escapejs-and-simplejson)